iib/workers/tasks/build.py

```diff
@@ -840,6 +888,36 @@ def handle_add_request(
             (prebuild_info['distribution_scope'] in ['dev', 'stage']),
         )
 
+        # Add the max ocp version property
+        # We need to ensure that any bundle which has deprecated/removed API(s) in 1.22/ocp 4.9
+        # will have this property to prevent users from upgrading clusters to 4.9 before upgrading
+        # the operator installed to a version that is compatible with 4.9
+
+        # Get the CSV name and version (not just the bundle path)
+        db_path = temp_dir + "/index.db"
```

Comment:
Just a heads up #250 is going to create a helper function which can be used here and will avoid code duplication. :)

Reply:
On second thoughts, do you need this piece of code, based on our conversation earlier, we only need to run this on bundles that we are adding in a request. So you can simply use the `bundles` variable that's coming in to apply the property on.

Reply:
in order to add the property I need the csv name and the version but bundles only gives me the pull spec. I figured instead of parsing the csv from the bundle I would just ask the db what these values are. What do you think?

Reply:
Yeah, I think that makes sense. `_get_missing_bundles ` is already calling grpcurl maybe you can do everything in one call instead of calling it again?

Reply:
it looks like the function is dependent on `_get_present_bundles` but this is being called before the new bundles (from the IIB request) are added to the index. I just need the name and version for the bundles from the IIB request that aren't in the index at that time unfortunately.